### PR TITLE
Make health client wo using it

### PR DESCRIPTION
**Pull request by hakonhall**
This PR should not have any functional changes.

 - Make a synthetic ApplicationInfo for the config server and "activate" it
   during bootstrap.
 - Make an ApplicationInstanceGenerator that converts an ApplicationInfo
   to an ApplicationInstance. Used by ModelGenerator.
 - Make a "DuperModel" that combines the SuperModel (deployed apps) with
   the set of synthetic apps.
 - Prefer health over Slobrok service status, if available (!= NOT_CHECKED).

Health monitoring is structured as:
 - A manager responsible for adding and removing applications
 - An ApplicationHealthMonitor responsible for all monitoring for a given app
 - A HealthMonitor responsible for monitoring the health of a single service
 - A HealthClient responsible for issuing health requests to fixed endpoint

Some of the lower-level health monitoring is not entirely finished, as it
depends on refactoring of an athenz dependency tbd later.

### CODE REVIEW

File: service-monitor/src/test/java/com/yahoo/vespa/service/monitor/internal/DuperModelTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            configServer1,
+            configServer2,
+            configServer3).collect(Collectors.toList());
+
+    private final ServiceStatusProvider statusProvider = mock(ServiceStatusProvider.class);
+
+    @Test
+    public void toApplicationInstance() throws Exception {
+        when(statusProvider.getStatus(any(), any(), any(), any())).thenReturn(ServiceStatus.NOT_CHECKED);
+        ConfigserverConfig config = new ConfigserverConfig(
```
Comment (freva) [inline service-monitor/src/test/java/com/yahoo/vespa/service/monitor/internal/DuperModelTest.java]: Use `ConfigserverUtil` ?

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/DuperModelListener.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.service.monitor.internal;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (freva) [inline service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/DuperModelListener.java]: Duplicate copyright

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/MonitorManager.java [hunk 3]
```diff
@@ -1,11 +1,10 @@
 // Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.service.monitor.internal;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (freva) [inline service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/MonitorManager.java]: Duplicate copyright

**Review by freva — APPROVED** [review]

File: service-monitor/src/test/java/com/yahoo/vespa/service/monitor/internal/DuperModelTest.java (hunk 1 above)
Comment (hakonhall) [reply service-monitor/src/test/java/com/yahoo/vespa/service/monitor/internal/DuperModelTest.java]: Done

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/DuperModelListener.java (hunk 2 above)
Comment (hakonhall) [reply service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/DuperModelListener.java]: Done

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/MonitorManager.java (hunk 3 above)
Comment (hakonhall) [reply service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/MonitorManager.java]: Done

**Review by freva — APPROVED** [review]

**Pull request merged**